e2e/paths/accessSegregation_test.js

```diff
@@ -0,0 +1,31 @@
+const config = require('../config.js');
+
+let caseId;
+
+Feature('Cases visible only to respective local authority and admin').retry(2);
+
+Before(async (I, caseViewPage) => {
+  I.logInAndCreateCase(config.swanseaLocalAuthorityEmailKurt, config.localAuthorityPassword);
+  caseId = await I.grabTextFrom('.heading-medium');
+  caseId = caseId.toString().replace('#', '');
+  caseViewPage.goToNewActions(config.applicationActions.submitCase);
+  I.click('Submit');
+  I.click('Sign Out');
+});
+
+Scenario('Different user in the same local authority can see case created', (I, loginPage, caseListPage) => {
+  loginPage.signIn(config.swanseaLocalAuthorityEmailDamian, config.localAuthorityPassword);
+  caseListPage.changeStateFilter('Submitted');
+  I.see(caseId);
```

Comment:
This test is fragile and will break when there is more than 25 submitted cases per environment.

Reply:
@geek-soft we happy to skip them?

Reply:
Nope, we should make them work. I remember the idea we had during 3 amigos session was to just jump to the case details url directly.